lblod/app-gn-publicatie
# Add migrations for new administrative units

### Overview
Copy over the relevant parts of the migrations to create new administrative units for the next iteration of government, from app-GN.

I was unable to test publishing an IV for a fusiegemeente as the IV can't be created for them. Publication for districts and normal gemeenten works (as well as display of previously published IVs).

##### connected issues and PRs:
Jira ticket: https://binnenland.atlassian.net/browse/GN-5211

### Setup
Needs to be connected to GN to pick up publications.

### How to test/reproduce
In GN, publish an IV. See that it get's picked up and is displayed in the search for that munipality.

### Challenges/uncertainties
There were a number of fixes in the data in app-GN. I tried to keep the parts that we need rather than just copying the whole lot.

### Checks PR readiness
